website/.husky/pre-commit

```diff
@@ -0,0 +1,3 @@
+cd website
+yarn fmt && git add .
```

Comment:
does the `.husky` need to be outsite the `website` folder?

Reply:
I was referring to their doc here: https://typicode.github.io/husky/how-to.html#project-not-in-git-root-directory